aten/src/ATen/native/native_functions.yaml

```diff
@@ -1791,6 +1791,7 @@
 # We need this to be able to properly copy from a CPU to an XLA tensor with different sizes.
 # See https://github.com/pytorch/xla/issues/2881
 - func: _copy_from_and_resize(Tensor self, Tensor dst) -> Tensor
+  device_check: NoCheck   # TensorIterator
```

Comment:
Based on the comment above, this is also used by XLA so we might want to be careful with removing that when it is used out of core.